BrowserKit/Sources/Common/Theming/Themeable.swift

```diff
@@ -3,55 +3,92 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import UIKit
+import Combine
 
-public protocol Themeable: ThemeUUIDIdentifiable, AnyObject {
-    var shouldUsePrivateOverride: Bool { get }
+public protocol Themeable: ThemeUUIDIdentifiable {
+    /// Whether we should override private theme whether we want to force the theme to be private or not private. Going
+    /// against the basic theme set up.
```

Comment:
Nit, there's two `Whether`. Hopefully the rewrite makes sense
```suggestion
    /// We should override the private theme setting only when we explicitly want to force the theme to be private or not private.
    /// Going against the basic theme set up.
```

Reply:
LOL This was copy and pasted from before, not from me 😆 But you're right I don't quite understand what this says. I'll rewrite!